src/arrays.js

```diff
@@ -7,6 +7,18 @@
  */
 const containsAll = (baseArr, arr) => arr.every((ele) => baseArr.includes(ele));
 
+/**
+ * Check that every element in an array exists in the other array.
```

Comment:
Nice function! Really Useful.

But, I guess the description needs to be enhanced, I bet it's the **containsAll** function's description.

Thanks.